softwareconfiguration/mlplan/full/src/main/java/ai/libs/mlplan/cli/MLPlanCLI.java

```diff
@@ -203,6 +229,15 @@ private static void runMLPlan(final CommandLine cl) throws Exception {
 			System.err.println("Cannot use both: local dataset and openml task. Only one option either " + O_FIT_DATASET + " or " + O_OPENML_TASK + " may be given.");

```

Comment:
syso ;)

Reply:
this is the CLI of ml-plan. I'd argue that for CLI this is better than a logger.